.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -72,18 +61,41 @@ jobs:
           EOM
             fi
           done
+
+      - name: Check issue.md existence
+        id: check_files
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "issue.md"
+      - name: Create issue
+        if: steps.check_files.outputs.files_exists == 'true'
+        run: |
+          # Heredoc for issue header
+          cat <<- EOM > issue-full.md
```

Comment:
Rather than introducing a new file, we could also try to prepend the header to the content of issue.md.

Logic in pseudo code:
issue.md = <header string> + issue.md (generated in previous steps)

Just not sure how to do this in a shell script :)